Arsenic-ATG/Qt-calculator
# Add new calculator theme

Description:
- This pull request updates the calculator application's theme by introducing a vibrant color scheme and reorganizing the button layout for a more intuitive user experience.

Changes Made:
- Updated CSS Styles:
  - Applied a new color scheme with three distinct button colors:
    - Pink: Clear and equal buttons.
    - Yellow: Number buttons.
    - Teal: Arithmetic and additional operation buttons.
  - The new colors improve visibility, making buttons easier to distinguish.
- Reorganized Button Layout:
  - Rearranged arithmetic operators and some additional buttons to align with the equal button.

How to Test:
- Build and run the application.
- Verify that buttons display the correct colors as described.
- Ensure that button functionality remains consistent after the layout adjustments.

Additional Notes:
- These changes improve the application's visual appeal and usability.
- Future themes can be easily added by extending the current CSS styles.
